program/dialog.c

```diff
@@ -77,7 +77,13 @@ bool DialogMain(ref Character)
 	Dialog.CurrentNode = CharacterRef.Dialog.CurrentNode;
 	startDialogMainCounter = 0;
 	SetEventHandler("frame", "StartDialogMain", 1);
+
 	SetTimeScale(0.0);
+	TimeScaleCounter = -3;
```

Comment:
Should be 0, because time after dialog is reset to 1x

Reply:
Thanks for reviewing. Should be fixed now.